client-cli/src/command/transaction_command.rs

```diff
@@ -1055,11 +1038,8 @@ fn ask_node_metadata() -> Result<CouncilNode> {
         ask("please enter base64 encoded keypackage:");
         match base64::decode(&text().chain(|| (ErrorKind::IoError, "Unable to read keypackage"))?) {
             Ok(kp) => {
-                if let Err(err) = verify_keypackage(&kp) {
-                    println!("invalid keypackage: {}", err);
-                } else {
-                    break kp;
-                }
+                /* TODO : use dev-utils to verify*/
```

Comment:
why not verify directly?

Reply:
because mls, ra-client moved to dev-utils, cannot call verification directly.

Reply:
i'm adding verification now